packages/sveltekit/package.json

```
@@ -20,6 +20,10 @@
     "./package.json": "./package.json",
     ".": {
       "types": "./build/types/index.types.d.ts",
+      "worker": {
+        "import": "./build/esm/index.worker.js",
+        "require": "./build/cjs/index.worker.js"
+      },
```

not directly applicable to this line but: I had to make this change to the `node` export to start the build:

```suggestion
  "node": {
    "require": "./build/cjs/index.server.js",
    "import": "./build/esm/index.server.js"
  }
```

However, even after this change, I get another error a bit later

---

Sorry @Lms24 if this isn't the right place to discuss this, but is there any reason why we don't make this change in a separate PR?

I'm having an issue where trying to import `import * as Sentry from '@sentry/sveltekit';` using ESM in Node.JS has all the `@sentry/node` changes under `Sentry.default`, and the above change seems to fix it.

From reading https://github.com/getsentry/sentry-javascript/pull/9872#discussion_r1430500153, it looks like ESM used to be broken, but I think https://github.com/getsentry/sentry-javascript/commit/7f2e804c6985e8bc4430d1763ec5aee577eec1ed might have fixed it? But a similar change for remix seems to have not worked: https://github.com/getsentry/sentry-javascript/pull/12742

---

@aloisklink sorry for only seeing your comment now: I agree, we should make this change separately. Thing is, as far as I know, SvelteKit transpiles to CJS (at least for Node-based environments). So I'm a bit hesitant to add an ESM entry point, especially because this works even less well than it currently works in CJS, with OpenTelemetry.

Feel free to submit a PR and let's see what our tests have to see